src/sentry/testutils/cases.py

```diff
@@ -438,3 +440,27 @@ class CliTestCase(TestCase):
     def invoke(self, *args):
         args += tuple(self.default_args)
         return self.runner.invoke(self.command, args, obj={})
+
+
+@pytest.mark.usefixtures('browser_class', 'percy_class')
+class AcceptanceTestCase(LiveServerTestCase):
+    # Use class setup/teardown to hold Selenium and Percy state across all acceptance tests.
+    # For Selenium, this is done for performance to re-use the same browser across tests.
+    # For Percy, this is done to call initialize and then finalize at the very end after all tests.
+    #
+    # TODO: if acceptance tests are split across files, this will need to be refactored into a
+    # pytest plugin/fixture or something else that can manage global state.
+
+    @before
+    def clear_cookies(self):
```

Comment:
couldn't figure out how to move this into a pytest fixture that only existed if browser was setup